src/close_fds.c

```diff
@@ -74,10 +73,8 @@ void close_other_fds()
 {
 	int fd;
 
-	for (fd = 3; fd < open_files_max_fd; fd++) {
+	for (fd = 3; fd <= open_files_max_fd; fd++) {
 		if (open_files_set == NULL || FD_ISSET(fd % FD_SETSIZE, &(open_files_set[fd / FD_SETSIZE])))
-			if (fd == sync_pipe_fd || fd == attach_pipe_fd || fd == dev_null_r || fd == dev_null_w || fd == oom_cgroup_fd
-			    || fd == oom_event_fd)
-				close(fd);
+			close(fd);
```

Comment:
do we change the name "close_other_fds" if we're closing all the fds?

Reply:
we are closing only the fds that were open when the program started so it is still the "other fds" that weren't opened by conmon itself

Reply:
I am not sure I understand how this fixes the problem. I can see the `<=` being helpful (we're not missing one of the fds). but why do we no longer ignore all of the fds conmon itself opened. Those are closed in different sections of the code AFAIK, and we'd double close them if we close them here, right?

Reply:
we were previously closing the fd if it was one of `sync_pipe_fd`, `attach_pipe_fd`, `dev_null_r`, `dev_null_w`, `oom_cgroup_fd`, `oom_event_fd` and that never triggered because these fds were open after the program started up.  The logic was broken since the beginning, although it was enough to fix the 100% cpu usage bug we had earlier